cms/djangoapps/contentstore/management/commands/tests/test_force_publish.py

```diff
@@ -0,0 +1,91 @@
+"""
+Tests for the force_publish management command
+"""
+import mock
+from django.core.management.base import CommandError
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from contentstore.management.commands.force_publish import Command
+from contentstore.management.commands.utils import get_course_versions
+
+
+class TestForcePublish(ModuleStoreTestCase):
+    """
+    Tests for the force_publish management command
+    """
+    def setUp(self):
+        super(TestForcePublish, self).setUp()
+        self.course = CourseFactory.create(default_store=ModuleStoreEnum.Type.split)
+        self.command = Command()
+
+    def test_no_args(self):
+        """
+        Test 'force_publish' command with no arguments
+        """
+        errstring = "force_publish requires 1 argument: <course_id>"
+        with self.assertRaisesRegexp(CommandError, errstring):
+            self.command.handle()
+
+    def test_invalid_course_key(self):
+        """
+        Test 'force_publish' command with invalid course key
+        """
+        errstring = "Invalid course key."
+        with self.assertRaisesRegexp(CommandError, errstring):
+            self.command.handle('TestX/TS01')
+
+    def test_too_many_arguments(self):
+        """
+        Test 'force_publish' command with more than 1 arguments
+        """
+        errstring = "force_publish requires 1 argument: <course_id>"
+        with self.assertRaisesRegexp(CommandError, errstring):
+            self.command.handle(unicode(self.course.id), 'invalid-arg')
+
+    def test_course_key_not_found(self):
+        """
+        Test 'force_publish' command with non-existing course key
+        """
+        errstring = "Course not found."
+        with self.assertRaisesRegexp(CommandError, errstring):
+            self.command.handle(unicode('course-v1:org+course+run'))
+
+    def test_force_publish_non_split(self):
+        """
+        Test 'force_publish' command doesn't work on non split courses
+        """
+        course = CourseFactory(default_store=ModuleStoreEnum.Type.mongo)
+        errstring = 'The owning modulestore does not support this command.'
+        with self.assertRaisesRegexp(CommandError, errstring):
+            self.command.handle(unicode(course.id))  # pylint: disable=E1101
```

Comment:
you won't need this anymore, but when disabling a pylint rule, do the human-readable version. So `disable=no-member` instead of `disable=E1101`. But, once you address my comment above, you won't need this comment anymore